modules/kn-trigger-update.adoc

```diff
@@ -1,43 +1,42 @@
 // Module included in the following assemblies:
 //
-// * serverless/event_workflows/serverless-kn-trigger.adoc
+// * /serverless/event_workflows/serverless-using-brokers.adoc
 
 [id="kn-trigger-update_{context}"]
 = Updating a trigger using `kn`
 
 You can use the `kn trigger update` command with certain flags to update attributes for a trigger.
 
-.Example
-
-. To update a trigger, enter the following command:
-+
-
+.Example command
 [source,terminal]
 ----
 $ kn trigger update <trigger_name> --filter <key=value> --sink <sink_name> [flags]
 ----
 
-+
-You can update a trigger to filter exact event attributes that match incoming events, such as `type=knative.dev.event`. For example:
-+
+[id="kn-trigger-update-examples_{context}"]
+== Updating trigger examples
+
+You can update a trigger to filter exact event attributes that match incoming events, such as `type=knative.dev.event`:
 
+.Example command
 [source,terminal]
 ----
 $ kn trigger update <trigger_name> --filter type=knative.dev.event
 ----
 
-. You can also remove a filter attribute from a trigger. For example, you can remove the filter attribute with key `type`:
-+
+You can also remove a filter attribute from a trigger.
+For example, you can remove the filter attribute with key `type`:
```

Comment:
I'm not sure why you bumped this sentence to its own line, but normally we'd keep sentences in a paragraph together on  one line instead. Not critical though.